compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs

```diff
@@ -325,6 +328,19 @@ mkConfFile PackageConfigFields {..} pkgModuleNames pkgId = do
             , "depends: " ++ unwords deps
             ]
 
+sinkEntryDeterministic
+    :: Zip.CompressionMethod
+    -> ConduitT () BS.ByteString (ResourceT IO) ()
+    -> Zip.EntrySelector
+    -> Zip.ZipArchive ()
+sinkEntryDeterministic compression sink sel = do
+    Zip.sinkEntry compression sink sel
+    Zip.setModTime fixedTime sel
+  -- The ZIP file format uses the MS-DOS timestamp format
+  -- (I didn’t even knew that existed) which starts at
+  -- 1980 rather than 1970.
```

Comment:
I was literally about to ask why not use the unix epoch. That explains it. Thanks.